Misc.cpp

```diff
@@ -10,7 +11,10 @@ HOOK_METHOD(CrewEquipBox, RemoveItem, () -> int)
     LOG_HOOK("HOOK_METHOD -> CrewEquipBox::RemoveItem -> Begin (Misc.cpp)\n")
     int ret = super();
     // TODO: R4V3ON Really wants this to be an option to disable in a custom options menu (or maybe in hyperspace.xml)
-    G_->GetSoundControl()->PlaySoundMix("airLoss", -1.f, false);
+    if (CustomOptionsManager::GetInstance()->dismissSound.currentValue==true)
+    {
+        G_->GetSoundControl()->PlaySoundMix(CustomOptionsManager::GetInstance()->dismissSound.currentString, -1.f, false);
```

Comment:
We probably want a default value for this so it remains backwards compatible with current Multiverse (since they use their own hyperspace.xml right now) so this currently would cause a breaking change as it would not be defined.

Reply:
Set default options in CustomOptions.h for backwards compatibility